app/views/welcome/index.html.erb

```diff
@@ -1,21 +1,19 @@
-<main class="welcome" role="main">
```

Comment:
what was wrong with "main"?

Reply:
There are 2 of them. There is a <main class="internal" role="main"> just outside this main on the welcome page. I believe this is from application.html.erb.